.github/actions/install-dependencies/action.yml

```diff
@@ -5,8 +5,14 @@ runs:
   steps:
   - shell: bash
     run: |
-      sudo apt-get update -y
-      sudo apt-get install -y ubuntu-dev-tools libbsd-dev
-      sudo sed -Ei 's/^# deb-src /deb-src /' /etc/apt/sources.list
-      sudo apt-get update -y
+      sudo cat /etc/apt/sources.list.d/ubuntu.sources
+      sudo sed -i '/deb-src/d' /etc/apt/sources.list.d/ubuntu.sources
+      sudo sed -i 's/\bdeb\b/deb deb-src/g' /etc/apt/sources.list.d/ubuntu.sources
+      export DEBIAN_PRIORITY=critical
+      export DEBIAN_FRONTEND=noninteractive
+      # let's try to work around upgrade breakage in a pkg we don't care about
+      sudo apt-mark hold grub-efi-amd64-bin grub-efi-amd64-signed
```

Comment:
This seems to import parts of other scripts so that you can merge them into a single one.  I think that would be better in a separate commit, after first fixing the CI problem.

Reply:
I'd prefer to keep it as it is

Reply:
Ok.

Reply:
Then the commit message should probably mention that.

Reply:
I added that to the commit message description.